snippets/card-product.liquid

```diff
@@ -30,16 +30,16 @@
       assign ratio = 1
     endif
   -%}
-  <div class="card-wrapper underline-links-hover">
+  <div class="product-card-wrapper card-wrapper underline-links-hover">
```

Comment:
Same suggestion as above but with `card-wrapper--product` instead of `product-card-wrapper`

```suggestion
  <div class="card-wrapper card-wrapper--product underline-links-hover">
```

We'd of course need to update the `base.css` selector from `.product-card-wrapper .card` to `.card-wrapper--product .card`